task/deprecated-image-check/0.1/deprecated-image-check.yaml

```diff
@@ -33,62 +30,66 @@ spec:
       image: registry.access.redhat.com/ubi8/ubi
       script: |
         #!/usr/bin/env bash
-        echo "Querying Pyxis for $(params.IMAGE_REPOSITORY)..."
-        http_code=$(curl -s -k -o $(workspaces.sanity-ws.path)/repository_data.json -w '%{http_code}' "https://catalog.redhat.com/api/containers/v1/repositories/registry/$(params.IMAGE_REGISTRY)/repository/$(params.IMAGE_REPOSITORY)")
-
-        echo "Response code: $http_code"
-        echo -n $http_code > $(results.PYXIS_HTTP_CODE.path)
+        export IMAGE_REPO_PATH=$(workspaces.sanity-ws.path)/${IMAGE_REPOSITORY}
+        IFS=$'\n' read -d'' -r -a IMAGE_ARRAY < <(echo "${params.BASE_IMAGES_DIGESTS}" | sed 's/\\n/\'$'\n''/g' && printf '\0')
```

Comment:
I think that `${params.BASE_IMAGES_DIGESTS}` should be `$(params.BASE_IMAGES_DIGESTS)` since it's a Tekton variable.